docs/source/api/plugin/landing-pages.mdx

```diff
@@ -459,97 +459,6 @@ The default value is `dark`.
 </tbody>
 </table>
 
-## GraphQL Playground landing page
```

Comment:
It might be worth leaving a brief section saying that AS3 had a GraphQL Playground landing page and we've done a one-time publish of a new package but otherwise don't support it (ie, no actual example or list of options or anything).

Reply:
[312add2](https://github.com/apollographql/apollo-server/pull/6925/commits/312add249c878dacd4115cb0670264c4dd15d2a6)